frontend/src/components/TransactionEventList/Events/DataRegistered.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-	<span>Data was registered</span>
+	<span>Data was registered "{{ event.dataAsHex }}"</span>
```

Comment:
We should be trying to do CBOR encoding here as well. Same as for memos.

Reply:
done